javascript/introduction-to-js-1/troubleshooting/number-game-errors.html

```diff
@@ -1,7 +1,7 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
For some reason my Prettier uppercases `doctype`, but it doesn't affect line or character numbers, so no matter.

Reply:
Huh, interesting. The VSCode extension does indeed capitalise it whereas running Prettier through a Neovim plugin or via their CLI (v3.3.3) is happy with it lowercase. But yeah, no matter since it doesn't cause any meaningful discrepancies, but odd nonetheless.